Update some of the openssh default config to disable passwords and add some liboqs specific settings

--- a/sshd_config
+++ b/sshd_config
@@ -53,7 +53,7 @@
 #IgnoreRhosts yes
 
 # To disable tunneled clear text passwords, change to no here!
-#PasswordAuthentication yes
+PasswordAuthentication no
 #PermitEmptyPasswords no
 
 # Change to yes to enable challenge-response passwords (beware issues with
@@ -95,7 +95,7 @@
 #TCPKeepAlive yes
 #PermitUserEnvironment no
 #Compression delayed
-#ClientAliveInterval 0
+ClientAliveInterval 180
 #ClientAliveCountMax 3
 #UseDNS no
 #PidFile /var/run/sshd.pid
